test/integration_test.go

```diff
@@ -148,18 +149,24 @@ func (ts *IntegrationTestSuite) SetupTest() {
 	ts.seq++
 	ts.activities.clearInvoked()
 	ts.taskListName = fmt.Sprintf("tl-%v", ts.seq)
-	ts.worker = worker.New(ts.rpcClient.Interface, domainName, ts.taskListName, worker.Options{
```

Comment:
huh.  so we were previously creating a worker and throwing it away?

thanks for the cleanup :+1:, surprising that got missed when it was introduced.

Reply:
test code changes usually get overlooked in reviews so I guess that's what happened here